src/Units/Unit.java

```diff
@@ -86,6 +86,28 @@ public UnitContext getRangeContext(GameMap map, WeaponModel weapon)
     return new UnitContext(map, this, weapon);
   }
 
+  public boolean capture(MapLocation target)
```

Comment:
Why move `capture` from `UnitState` to `Unit`?

Reply:
There's no `UnitContext` constructor that takes a `UnitState`, and I didn't feel like making one.

Reply:
Fair enough I guess. This PR is pretty straightforward otherwise, so fire when ready.